content/positions/data-infrastructure-engineering-intern/index.md

```diff
@@ -0,0 +1,58 @@
+
+A strong background in software engineering is required. A specific focus on open source development and data infrastructure is desired. An understanding of the technical concepts underlying cloud-based workflow orchestration and databases may be helpful, but more important is an ability to learn and interpret the documentation for these tools. Members of communities traditionally underrepresented in software engineering or ecology are strongly encouraged to apply.
+
+## Hours, dates, and work location
+The preferred start date is late October 2024. The position will extend through at least May 2025, with an option for extension into the summer and beyond based on funding and performance. There will be 10-20 hours of work per week depending on the intern's preference. The position may be remote or in person (office on the UC Davis campus) depending on preference.
+
+## Compensation
+Approximately $18.50 / hour.
+
+## Primary duties
+* Develop drone imagery processing workflows using Argo Workflows on cloud-based Kubernetes clusters, accommodating the evolving needs of the lab's drone imagery processing workflow.
+* Set up a PostgreSQL database in the project's cloud computing environment and integrate it into data processing workflows.
+* Develop scripted workflows for transferring processed data to the CyVerse Data Store in a carefully organized and thoroughly documented manner.
+* Document problems through github issues and respond to issues created by the supervisor and the community.
+* Propose and discuss improvements to efficiency or user experience.
+* Improve project-level documentation and example code.
+* Improve the quality of existing codebases by reorganizing, modularizing, documenting, and streamlining.
+* The intern will receive mentoring on general software development principles as well as feedback on their code through code reviews and meetings.
+
+## Minimum qualifications
+* Undergraduate student in software engineering, computer science, data science, or related field OR another field with demonstrated understanding of software engineering fundamentals
+* Experience with Python
```

Comment:
I do think Python would be useful but I'm not sure it's a requirement. At least for the types of things Avnoor worked on that we explicitly called infra tasks. Do you think the next phase will be substantially different?